app/streaming/input/mouse.cpp

```diff
@@ -55,6 +55,14 @@ void SdlInputHandler::handleMouseButtonEvent(SDL_MouseButtonEvent* event)
             return;
     }
 
+    if (m_SwapMouseButtons) {
+        if (button == SDL_BUTTON_RIGHT)
+            button = SDL_BUTTON_LEFT;
+        else if (button == SDL_BUTTON_LEFT)
+            button = SDL_BUTTON_RIGHT;
```

Comment:
These need to be using the Moonlight button definitions, not SDL's (see the switch statement above). This only works by luck (SDL's `SDL_BUTTON_LEFT` is the same value as `BUTTON_LEFT`)

Reply:
I have now updated the code so it uses Moonlights button definitions.